src/source/PeerConnection/Rtcp.c

```diff
@@ -306,7 +306,7 @@ STATUS onRtcpTwccPacket(PRtcpPacket pRtcpPacket, PKvsPeerConnection pKvsPeerConn
         // time not yet set (only happens for first rtp packet)
         localStartTimeKvs = twcc->twccPacketBySeqNum[(UINT16) sn].localTimeKvs;
     }
-    for (seqNum = sn; duration < TWCC_ESTIMATOR_TIME_WINDOW && seqNum != twcc->lastReportedSeqNum; seqNum++) {
+    for (seqNum = sn; seqNum != twcc->lastReportedSeqNum; seqNum++) {
```

Comment:
I am not very familiar with this piece. What's the intention here as this changes the logic? Some information in the PR might be useful. This file is in general poorly commented :(

Reply:
I agree. Twcc needs much better documentation.